test/lib/ansible_test/_data/injector/virtualenv.sh

```diff
@@ -1,8 +1,17 @@
 #!/usr/bin/env bash
 # Create and activate a fresh virtual environment with `source virtualenv.sh`.
 
+# Privilege venv if it is available but fallback to virtualenv
```

Comment:
For backwards compatibility I'd fallback to `venv` instead:

```suggestion
# Prefer virtualenv if it is available but fallback to venv.
```

Code changes to match will also be needed.

Reply:
Works for me.